app/src/main/java/com/onegini/mobile/exampleapp/view/activity/AuthenticationActivity.java

```diff
@@ -44,8 +45,8 @@
   @SuppressWarnings({ "unused", "WeakerAccess" })
   @Bind(R.id.authenticator_message)
   TextView authenticatorMessage;
-
-  @SuppressWarnings({"unused"})
+  @Nullable
```

Comment:
Currently all layout activities have this button (i.e. push with fingerprint) even if it's hidden and non-used. Maybe we should add it for "simple push" too, to make the implementation similar?